### ISSUE TITLE
MapTool crashes if unable to connect to sentry.io

### ISSUE THREAD

**Issue by kayila** [JamzTheMan/MapTool#78]
MapTool seems to crash if it can't connect to sentry.io. This is on ubuntu linux with Java 11

The output is here:
https://gist.github.com/kayila/731fc61f6df72a536b428a24dd013b4b

**Comment by JamzTheMan** [comment]
Yes, Oracle 10 is required. You can also install the .deb which packages it's own jre for use.
